hedera-evm/src/main/java/com/hedera/evm/model/Id.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.evm.model;
+
+import com.hederahashgraph.api.proto.java.AccountID;
+import org.hyperledger.besu.datatypes.Address;
+
+public record Id(long shard, long realm, long num) {
+
+    // TODO: FIX THIS
+    public Address asEvmAddress() {
+        return Address.fromHexString("TODO");
+    }
+
+    public static Id fromGrpcAccount(final AccountID id) {
```

Comment:
If use `Address` instead of `AccountID` for `fundingAccount`, we can get rid of this method.